scripts/deploy-olm.sh

```diff
@@ -74,6 +74,8 @@ fi
 if $SCRIPT_DIR/is-openshift.sh
 then
     OLM_NS=openshift-marketplace
+    # Add custom scc to serviceaccount
+    oc adm policy add-scc-to-user -n $NAMESPACE -z verticadb-operator-controller-manager anyuid-extra
```

Comment:
When do we create the anyuid-extra scc?  There should be an apply somewhere.

Reply:
Yes there was a script (create-custom-scc.sh) that was called in setup-olm.sh and I deleted it. As we only need to create the scc once, I could add an apply on the scc manifest, where the script was once called.

Reply:
Yes, lets do that.  The less manual steps the better.